apps/mainsite/seeds/02_users.py

```diff
@@ -4,10 +4,9 @@
 from allauth.account.models import EmailAddress
 from staff.models import InstitutionStaff
 
-
 # Institution
 [
-    Institution.objects.get_or_create(name=ins) for ins in [
+    Institution.objects.get_or_create(identifier=ins, name=ins) for ins in [
```

Comment:
What's the use of the identifier if it matches the name?

Reply:
identifier will be used to match users with institutions, name is any name they give it. i just populate it with id the first time i see it